ports/esp32s2/components/bootloader/subproject/main/bootloader_start.c

```diff
@@ -533,8 +533,8 @@ static void board_led_on(void)
 static void board_led_off(void)
 {
 #ifdef NEOPIXEL_PIN
-  delay_cycle( NEOPIXEL_RESET_DELAY ) ;
   board_neopixel_set(NEOPIXEL_PIN, RGB_OFF);
+  delay_cycle( NEOPIXEL_RESET_DELAY ) ;
 
   // Neopixel reset time
```

Comment:
the delay should be below its comment. But I push update for this change already.